src/i18n/en.js

```diff
@@ -35,6 +35,13 @@ export default {
     send: `Send`,
     show: `Show`,
   },
+  default_banner_data: {
+    background_url: 'https://qa.fromdoppler.com//wp-content//uploads//2019//06//violet-yellow.png',
+    description: 'my description',
+    functionality: 'my functionality',
+    image_url: 'https://qa.fromdoppler.com//wp-content//uploads//2019//06//login-es.746bf048.png',
```

Comment:
Remember to update these values when the final ones are ready.

Reply:
¿Que te parece utilizar los valores actuales como valores por defecto asi no necesitamos ninguna confirmación? luego, si los quieren cambiar, se cambian.

Reply:
Todavía está pendiente actualizar la URL de está imagen, no?